suite-native/graph/src/components/TransactionEventTooltip.tsx

```diff
@@ -19,6 +27,8 @@ type EventTooltipRowProps = {
     signValue: SignValue;
     value: number;
     networkSymbol: NetworkSymbol;
+    tokenAddress?: TokenAddress;
+    accountKey: AccountKey;
```

Comment:
nit: optional should be at the end.
```suggestion
    accountKey: AccountKey;
    tokenAddress?: TokenAddress;
```